client/main/main.js

```diff
@@ -219,39 +220,72 @@ $(document).ready(async() => {
         window.$body.addClass('electron');
 
 
-    Keyboard.addKeyboardListener('F10', () => host.toggleDevTools(), 'Toggle Developer Tools');
-    Keyboard.addKeyboardListener('F9',  () => instance.restartEngines(), 'Restart jamovi engines');
-    Keyboard.addKeyboardListener('Ctrl+KeyS', () => ActionHub.get('save').do(), 'Save current workspace');
-    Keyboard.addKeyboardListener('Ctrl+KeyO', () => ActionHub.get('open').do(), 'Open data file');
+    Keyboard.addKeyboardListener('F10', () => host.toggleDevTools(), _('Toggle Developer Tools'));
+    Keyboard.addKeyboardListener('F9',  () => instance.restartEngines(), _('Restart jamovi engines'));
```

Comment:
we typically don't really want people to know these exist ... so i wouldn't worry about translating them.